### DISCUSSION TITLE
Welcome to dbt-databricks Discussions!

### DISCUSSION THREAD

**Discussion by ueshin** [databricks/dbt-databricks#202] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by durbinar** [comment]
Hello Team, what are security measures implemented to protect the connector con. details, e.g. PAT,  ?

**Comment by jeffrey-harrison** [comment]
Hello, is there currently a way to find the databricks job-id or run-id for submitted jobs? Particularly when submitting with `submission_method: job_cluster`

We would like to link the created job in our pipelines and additionally modify permissions from the service principle so we can view those runs as any engineer. Our logs go through databricks sql session and don't seem to have any job-id or run-id.

![Screenshot 2023-08-17 at 11 18 31 AM](https://github.com/databricks/dbt-databricks/assets/136381696/a4ed8cb7-c1ea-42eb-a41f-4815daaa3065)

![Screenshot 2023-08-17 at 11 09 26 AM](https://github.com/databricks/dbt-databricks/assets/136381696/5764b008-8880-4794-8eb6-249462e5a339)

I believe I found where this information is:
https://github.com/databricks/dbt-databricks/issues/420
